src/main/java/de/embl/cba/mobie2/plot/ScatterPlotViewer.java

```diff
@@ -363,6 +368,7 @@ public void hide()
 	{
 		if ( window != null )
 			window.setVisible( false );
+		showScatterPlot = false;
```

Comment:
@K-Meech I think we don't need the `showScatterPlot`, we could just do `((window != null) && window.isVisible())` to check for visibility. 
And instead of `getShowScatterPlot` I would rename to `isVisible()`